src/components/forms/sections/WinterBoatDetails.tsx

```diff
@@ -10,15 +10,39 @@ import WinterUnRegisteredBoat from '../tabs/WinterUnRegisteredBoat';
 import { WinterStorageMethod } from '../../../__generated__/globalTypes';
 import { WinterFormValues } from '../../../types/winterStorage';
 import { WithBoatType } from '../Selects';
-import SectionSelector from './SectionSelector';
+import SectionSelector, { TypeProps } from './SectionSelector';
 
 type Props = {
   values?: WinterFormValues;
   tab: string;
+  requireBoat?: boolean;
+  showStorageMethod?: boolean;
 } & WithBoatType;
 
-const BoatDetails = ({ values, tab, boatTypes }: Props) => {
+const BoatDetails = ({ values, tab, requireBoat, boatTypes, showStorageMethod = true }: Props) => {
```

Comment:
Could we rename this to `WinterBoatDetails`?

Reply:
Yes! ✏️